deploy/siarnaq/main.tf

```diff
@@ -138,15 +138,15 @@ resource "google_cloud_run_service" "this" {
   template {
     spec {
       service_account_name  = google_service_account.this.email
-      container_concurrency = 16
+      container_concurrency = 4
 
       containers {
         image = var.image
 
         resources {
           limits = {
-            cpu    = "8000m"
-            memory = "4Gi"
+            cpu    = "2000m"
+            memory = "1Gi"
```

Comment:
could we only halve these? (feel a little unconfident about this, especially looking at current metrics)

Reply:
These graphs seem to be pretty far away from 25%. In addition, we're also reducing the number of concurrent requests per machine, which further alleviates.

<img width="467" alt="image" src="https://user-images.githubusercontent.com/16425187/212448675-412a27fc-0761-4c87-a2d0-09ac84881164.png">
<img width="447" alt="image" src="https://user-images.githubusercontent.com/16425187/212448685-e4121006-2354-44cc-a966-43b6d9c1301d.png">

Reply:
oh i think i misread graphs
i read the graphs as -- eg for bottom one -- "container utilized >=99% of memory, 11.99% of time"

but that would imply it spends more time at 99% than at 95% 
also that's, like,unintuitive and statistician-esque
I'll just trust you for now